packages/selenium-driver/src/smartui.js

```diff
@@ -17,8 +17,9 @@ async function smartuiSnapshot(driver, name, options = {}) {
             url: document.URL
         }), {});
 
-        await utils.postSnapshot({url, name, dom, options}, pkgName);
+        let { body } = await utils.postSnapshot({url, name, dom, options}, pkgName);
         log.info(`Snapshot captured: ${name}`);
+        if (body.data.warnings?.length !== 0) body.data.warnings.map(e => log.warn(e));
```

Comment:
add a check for , `if (body && body.data  && body.data.warnings?.length !== 0)`

Reply:
Resovled